extensions/stackdriver/stackdriver.h

```diff
@@ -73,7 +73,8 @@ class StackdriverRootContext : public RootContext {
   wasm::common::NodeInfo local_node_info_;
 
   // Indicates the traffic direction relative to this proxy.
-  PluginDirection direction_ = PluginDirection::Unspecified;
+  envoy::api::v2::core::TrafficDirection direction_ =
```

Comment:
I agree this is unnecessary coupling of extension code with xds api.

Reply:
Traffic direction is part of an API, so the question is whether we want to maintain an independent but coupled definition of it in WASM or just use xDS. I'd assume the second is better, since this is bound to evolve (the gateway option still needs to be added as a traffic direction).
Physically, we don't have to mandate protobufs. The client can just treat it as an int, and use a well-known value to compare.

Reply:
Can we use the original plugin direction for now? Maybe add a todo for it? There are several other duplication of envoy proto in WASM sdk such as grpc service (envoyproxy/envoy-wasm#17). For sure we need to remove them, but unless bazel is supported, there is no easy way to get around this duplication I think..

Reply:
The plugin direction enum is gone I think. I can add an enum here to reinterpret the int value.